### ISSUE TITLE
Why the perfomance of internvl2-8B is not good at internvl2-2B

### ISSUE THREAD

**Issue by zyandtom** [OpenGVLab/InternVL#1042]
same data, full parameter SFT, both use official 2nd_finetune script...Is there any change need to be done for 8B?

**Comment by Weiyun1025** [comment]
It seems to be an issue with overfitting; you might try lowering the learning rate. Additionally, [InternVL3.5](https://huggingface.co/papers/2508.18265) has been released, and we encourage you to try our newest models, which we believe will yield further performance improvements.

**Comment by zyandtom** [comment]
> It seems to be an issue with overfitting; you might try lowering the learning rate. Additionally, [InternVL3.5](https://huggingface.co/papers/2508.18265) has been released, and we encourage you to try our newest models, which we believe will yield further performance improvements.

请问3.5的sft代码库现在已经支持了吗
